app/controllers/financialStatement/ChargeDetailsController.scala

```diff
@@ -45,42 +48,49 @@ class ChargeDetailsController @Inject()(identify: IdentifierAction,
     with I18nSupport
     with NunjucksSupport {
 
-  def onPageLoad(identifier: String, startDate: LocalDate, chargeReference: String): Action[AnyContent] = identify.async {
+  def onPageLoad(identifier: String, startDate: LocalDate, index: String): Action[AnyContent] = identify.async {
```

Comment:
I'd call this chargeReferenceIndex to make it clear

Reply:
Done